egs/librispeech/WSASR/README.md

```diff
@@ -0,0 +1,205 @@
+# Introduction
+
+This is a weakly supervised ASR recipe for the LibriSpeech (clean 100 hours) dataset. We train a
+conformer model using [Bypass Temporal Classification](https://arxiv.org/pdf/2306.01031.pdf) (BTC)/Omni-temporal Classification (OTC) with transcripts with synthetic errors. In this README, we will describe
+the task and the BTC/OTC training process.
+
+Note that OTC is an extension of BTC and supports all BTC functions. Therefore, in the following, we only describe OTC.
+## Task
+We propose BTC/OTC to directly train an ASR system leveraging weak supervision, i.e., speech with non-verbatim transcripts. This is achieved by using a special token $\star$ to model uncertainties (i.e., substitution errors, insertion errors, and deletion errors) 
+within the WFST framework during training.
+
+
+<div style="display: flex;flex; justify-content: space-between">
+  <figure style="flex: 2; text-align: center; margin: 5px;">
+    <img src="figures/sub.png" alt="Image 1" width="25%" />
+
+  </figure>
+  <figure style="flex: 2; text-align: center; margin: 5px;">
+    <img src="figures/ins.png" alt="Image 2" width="25%" />
+
+  </figure>
+  <figure style="flex: 2; text-align: center;margin: 5px;">
+    <img src="figures/del.png" alt="Image 3" width="25%" />
+
+  </figure>
+</div>
+<figcaption> Examples of errors (substitution, insertion, and deletion) in the transcript. The grey box is the verbatim transcript and the red box is the inaccurate transcript. Inaccurate words are marked in bold.</figcaption> <br><br>
+
+
+We modify $G(\mathbf{y})$ by adding self-loop arcs into each state and bypass arcs into each arc. 
+  <p align="center">
+    <img src="figures/otc_g.png" alt="Image Alt Text" width="50%" />
```

Comment:
For this figure, there should be no by-pass arcs for arcs entering the last final state.

In k2, all arcs entering the final state have label == -1.

Reply:
Thanks. Updated.